packages/tatum-ada/src/wallet/wallet.ts

```diff
@@ -13,11 +13,10 @@ export const generateAdaWallet = async (mnemonic: string): Promise<WalletWithMne
 
 /**
  * Generate wallet
- * @param options.mnemonic mnemonic seed to use. If not present, new one will be generated
+ * @param mnemonic mnemonic seed to use. If not present, new one will be generated
  * @returns wallet or a combination of address and private key
  */
-export const generateWallet = (options: { mnemonic?: string }) => {
-  const mnemonic = options?.mnemonic
-  const mnem = mnemonic ? mnemonic : generateMnemonic(256)
-  return generateAdaWallet(mnem)
+export const generateWallet = (mnemonic: string) => {
```

Comment:
was optional

Reply:
This is required in master, the diff seems to be agains previous commit, not the original version.
https://github.com/tatumio/tatum-js/blob/d1de9b125a1b03ed72a6b3676dcadde9bb64891b/src/wallet/wallet.ts#L310